package.json

```diff
@@ -1531,6 +1536,11 @@
                     "command": "gitlens.stashSave",
                     "when": "gitlens:enabled",
                     "group": "2_gitlens@1"
+                },
+                {
+                    "command": "gitlens.externalDiff",
+                    "when": "gitlens:enabled",
+                    "group": "3_gitlens@1"
```

Comment:
I want move the `gitlens.openChangedFiles` and `gitlens.closedUnchangedFiles` into the `2_gitlens@1` and `2_gitlens@2` groups respectively, so can you change this one to `2_gitlens@3`. Also `gitlens.stashSave` should move to `3_gitlens@1`